app/components/school_group_comparison_component/school_group_comparison_component.html.erb

```diff
@@ -0,0 +1,85 @@
+<div class="pb-2">
+  <strong><%= t("advice_pages.#{@advice_page_key}.page_title") %></strong>
+  <p><%= t("school_groups.comparisons.how_does_it_compare.#{@advice_page_key}", default: '') %></p>
+  <div
+    id="<%= @id %>"
+    class="school-group-comparison-component-callout-row row mt-4">
+    <% SchoolGroupComparisonComponent::CATEGORIES.each do |category| %>
```

Comment:
I think you could inline some of this code into the component, by adding extra methods that are called by the template, e.g

```
<% categories.each do |category, count| %>
<% end %>
```

Where `categories` is a method on the component that yields the category and a count.